bigtable/bigtable.sh

```diff
@@ -243,6 +261,9 @@ function install_hbase() {
 }
 
 function main() {
+  if [[ ${OS_NAME} == debian ]] && [[ $(echo "${DATAPROC_IMAGE_VERSION} <= 2.1" | bc -l) == 1 ]]; then
```

Comment:
OS_NAME is not defined in this script, therefore it will fail. Also, please ensure that /gcbrun runs fine.